Library/ViewModels/PostCampaignCheckoutViewModelTests.swift

```diff
@@ -12,11 +12,14 @@ final class PostCampaignCheckoutViewModelTests: XCTestCase {
     Never
```

Comment:
Can you add more tests for the processing view in here?

Reply:
Yeah, of course! I wanted to make sure we were on the same page about going with this approach before I wrote proper tests. But I think we're good now!